windows/setup_py2exe.py

```diff
@@ -0,0 +1,296 @@
+    os.makedirs('dist')
+
+logger.info('Copying GTK files and icon')
+shutil.copyfile( GTK_DIR + '\\bin\\intl.dll',  'dist\\intl.dll')
+
+shutil.copytree(GTK_DIR + '\\etc', 'dist\\etc')
+shutil.copytree(GTK_DIR + '\\lib', 'dist\\lib')
+shutil.copytree(GTK_DIR + '\\share', 'dist\\share')
+shutil.copyfile( 'bleachbit.png',  'dist\\share\\bleachbit.png')
+
+
+#logger.info('Compressing executables')
+#files = recursive_glob('dist', ['*.pyd', '*.dll', '*.exe'])
+#for file in files:
+#    compress(UPX_EXE, UPX_OPTS, file)
+
+
+logger.info('Purging unnecessary GTK+ files')
+cmd= sys.executable +  ' setup.py clean-dist'
+run_cmd(cmd)
+
+
+logger.info('Copying BleachBit localizations')
+shutil.rmtree('dist\\share\\locale', ignore_errors=True)
+shutil.copytree('locale', 'dist\\share\\locale')
+assert_exist( 'dist\\share\\locale\\es\\LC_MESSAGES\\bleachbit.mo' )
+
+logger.info('Copying BleachBit cleaners')
+if not os.path.exists( 'dist\\share\\cleaners'):
+    os.makedirs( 'dist\\share\\cleaners')
+cleaners_files = recursive_glob('cleaners', ['*.xml'])
+for file in cleaners_files:
+    shutil.copy( file,  'dist\\share\\cleaners')
+
+
+logger.info('Checking for CleanerML')
+assert_exist( 'dist\\share\\cleaners\\internet_explorer.xml' )
+
+
+logger.info('Checking for Linux-only cleaners')
+if os.path.exists( 'dist\\share\\cleaners\\wine.xml'):
+    logger.info('grep -l os=.linux. dist/share/cleaners/*xml | xargs rm -f')
+
+
+
+logger.info('Signing code')
+if os.path.exists('CodeSign.bat') :
+    cmds = ['CodeSign.bat dist\\bleachbit.exe', 'CodeSign.bat dist\\bleachbit_console.exe']
+    for c in cmds:
+        run_cmd(c)
+else:
+    logger.error('CodeSign.bat not found')
+
+
+
+if not fast:
+    logger.info('Recompressing library.zip with 7-Zip')
+    if not os.path.exists( SZ_EXE ) :
+        logger.warning(SZ_EXE + ' does not exist')
+    else:
+        if not os.path.exists( 'dist\\library' ):
+            os.makedirs( 'dist\\library' )
+        cmd = SZ_EXE + ' x  dist\\library.zip' + ' -odist\\library  -y'
+        logger.info( cmd )
+        run_cmd(cmd)
+        file_size = os.path.getsize( 'dist\\library.zip' ) / (1024*1024.0)
+        logger.info( 'Size before 7zip recompression ' + str( file_size ) + ' Mb')
+        shutil.rmtree('dist\\library.zip', ignore_errors=True)
+
+        cmd = SZ_EXE + '  a -tzip -mx=9 -mfb=255 ..\\library.zip'
+        logger.info( cmd )
+        p = subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd='dist\\library')
+        stdout, stderr = p.communicate()
+        logger.info(stdout)
+        if stderr :
+            logger.error(stderr)
+
+        file_size = os.path.getsize( 'dist\\library.zip' ) / (1024*1024.0)
+        logger.info( 'Size after 7zip recompression ' + str( file_size ) + ' Mb')
+        shutil.rmtree( 'dist\\library', ignore_errors=True )
+        assert_exist( 'dist\\library.zip')
+
+        logger.info('Building portable')
```

Comment:
I can see you carefully followed `setup_py2exe.bat`, but it seems to have a bug. If you don't mind, please build the portable regardless of the setting of the variable `fast`